plutus-contract/src/Plutus/Contract.hs

```diff
@@ -64,10 +79,14 @@ module Plutus.Contract(
     , tell
     -- * Transactions
     , WalletAPIError
+    , Request.submitTxOld
     , Request.submitTx
     , Request.submitTxConfirmed
+    , Request.submitTxConstraintsOld
     , Request.submitTxConstraints
+    , Request.submitTxConstraintsSpendingOld
     , Request.submitTxConstraintsSpending
+    , Request.submitTxConstraintsWithOld
```

Comment:
What about using of qualified imports and moving old functions to a different module instead of `Old` suffix?

Reply:
I'll be removing the functions with the `Old` suffix in this PR.